### DISCUSSION TITLE
PVP balance

### DISCUSSION THREAD

**Discussion by Bredley** [chromiecraft/chromiecraft#3243] [Suggestions]
Hello,

a lot of talking was happening in BGs about balance but no actions are happening.

I would love to open a ticket for PVP balance. All i know now is that there is flat 20% dmg reduction. I wouldnt mind getting a higher reduction and healing reduction also, especially with BWL and further raids comming. 

The things i noticed so far that are most OP:
- BM hunters pets. They are hella fast, hella strong - even me as prot warrior getting shredded in few seconds.
- Not sure about all healers but Dpriest seems kinda unkillable  - one panance heals almost everyone full in 2sec
- IDK how much is it possible to balance matchmaking but some WSG are way too one-sided. Getting no heal vs 3 healers / 5 hunters vs 0 hunters, etc etc

**Comment by enzospin360** [comment]
I agree about the BM hunter pets.  I can't even 1v1 them and they crit me as a plate user for 900.  No pet should be able to crit 900.

**Comment by Spook9** [comment]
BG 20% dmg reduction needs to be upped to 30% or around that. Currently there is little counterplay and skill required due to the current burstyness and insane damage. Even healing reduction might be needed, just an idea so take it as you wish, but remember to add 🧂.

Damage will only keep scaling up as we progress into BWL and AQ so some more reduction will be needed and asked for by the community as it should be.
